htslib/kstring.h

```diff
@@ -144,7 +148,7 @@ static inline void ks_initialize(kstring_t *s)
 /// Resize a kstring to a given capacity
 static inline int ks_resize(kstring_t *s, size_t size)
 {
-	extern HTSLIB_EXPORT int ks_resize2(kstring_t *s, size_t size);
+	HTSLIB_EXPORT int ks_resize2(kstring_t *s, size_t size);
```

Comment:
Did `extern` here need to be removed because it caused a problem for one of the various flavours of `HTSLIB_EXPORT`?

Reply:
I see now [why](https://docs.microsoft.com/en-us/cpp/cpp/definitions-and-declarations-cpp?view=vs-2019) it was needed. I'll add it back.